### DISCUSSION TITLE
Not able to run test on GitHub Actions CI

### DISCUSSION THREAD

**Discussion by haxandev** [DevSkillsHQ/discusssions#40] [Help]
Hello guys, I'm using Laravel boilerplate, and locally test cases are passing, it seems like it is a database connection issue with SQLite on CI because on CI those test cases are working which are without the connection of the database. Really appreciate it If anyone could help on this matter.

 Here is my package.json file.
<img width="1408" alt="Screenshot 2023-09-18 at 8 36 25 AM" src="https://github.com/DevSkillsHQ/discusssions/assets/28425463/6993172a-fab6-4a4d-92b8-8b3d792be663">

**Comment by fenske** [comment]
Hi @haxandev. You can reproduce the failure by spinning up a Gitpod env (see README).

Wild guess: could it be that you forgot to commit your .env file?
